api/v1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1 contains API Schema definitions for the e2etest.grpc.io v1 API group
+// +kubebuilder:object:generate=true
+// +groupName=e2etest.grpc.io.e2etest.grpc.io
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "e2etest.grpc.io.e2etest.grpc.io", Version: "v1"}
```

Comment:
This can't be right.

```suggestion
	GroupVersion = schema.GroupVersion{Group: "e2etest.grpc.io", Version: "v1"}
```